src/Microsoft.ML.Tokenizers/Model/SentencePieceBaseModel.cs

```diff
@@ -59,6 +59,67 @@ internal SentencePieceBaseModel(ModelProto modelProto, bool addBos = false, bool
                                 specialTokens);
         }
 
+        internal SentencePieceBaseModel(SentencePieceOptions options)
```

Comment:
Is there any value in translating the protobuf type to this type to make a single initialization path?

Reply:
This will have perf overhead so I better avoid that.